src/test/java/com/jcabi/aether/ClasspathTest.java

```diff
@@ -94,6 +94,35 @@ public void buildsClasspath() throws Exception {
     }
 
     /**
+     * Classpath can build a classpath with more scopes.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void buildsClasspathWithMoreScopes() throws Exception {
+        MatcherAssert.assertThat(
+            new Classpath(
+                this.project(
+                    this.dependency(
+                        ClasspathTest.GROUP, ClasspathTest.GROUP, "4.12"
```

Comment:
@amihaiemil can we reduce duplication between this test and others that are similar?